app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeViewModel.kt

```diff
@@ -223,6 +244,10 @@ class ComposeViewModel @Inject constructor(
         return modifiedInitialState || textChanged || contentWarningChanged || mediaChanged || pollChanged || didScheduledTimeChange
     }
 
+    private fun isEmpty(content: String?, contentWarning: String?): Boolean {
```

Comment:
I'm a little concerned about this function.

It takes two parameters, but it operates on five values in total (`modifiedInitialState`, `media`, and `poll` being the others). So a casual reading of the code could lead to confusion.

I don't see a particularly clean way to fix that without a considerable rewrite of the viewmodel, so I'm mentioning this in case you can think of something better. If not, there's a rewrite coming, so it's not a big issue if this goes in as is.

Reply:
I basically follow `didChange` but I agree that it is a bit confusing.
That said I don't see a good solution for this either. If we already have a plan for a rewrite, I'd leave this as is for now.